src/tools/rustfmt/tests/target/anonymous-types.rs

```diff
@@ -16,4 +16,16 @@ struct Foo {
     e: f32,
 }
 
+// Test for issue 117942
```

Comment:
can you provide the full link to the rust-lang/rust issue in the comment. It'll help make it clearer that this isn't referring to a rustfmt issue.